MoltenVK/MoltenVK/GPUObjects/MVKDescriptorSet.mm

```diff
@@ -249,15 +259,23 @@
 	uint32_t bindCnt = (uint32_t)layout->_bindings.size();
 	for (uint32_t bindIdx = 0; bindIdx < bindCnt; bindIdx++) {
 		MVKDescriptorSetLayoutBinding* mvkDSLBind = &layout->_bindings[bindIdx];
-		uint32_t descCnt = mvkDSLBind->getDescriptorCount();
```

Comment:
Given your change above to `getDescriptorCount()`, could this function remain unchanged, and rely on the count always being `1` in the case of `VK_DESCRIPTOR_TYPE_INLINE_UNIFORM_BLOCK_EXT`?

Reply:
It can't for two reasons:
* `getDescriptorIndex()` needs `0` as second argument (not `pDescriptorAction->dstArrayElement`)
* `_descriptors[dstStartIdx + descIdx]->write()` needs `pDescriptorAction->dstArrayElement` as third argument (not `descIdx`)

So the if branch of `VK_DESCRIPTOR_TYPE_INLINE_UNIFORM_BLOCK_EXT` uses the parameters differently than the else loop does. The same goes for `read()`.